pages/index.tsx

```diff
@@ -13,11 +19,19 @@ const Home: NextPage = () => {
       <main>
         <h1>Wie schnell kannst du lesen?</h1>
         <p>
-          Ein(e) <b>"Schnellleser(in)"</b> verschlingt Bücher und kann sich
-          trotzdem noch an Details erinnern. <br /> Eine echte <b>Superpower</b>{" "}
-          <br /> <b>Hast du sie auch?</b>
+          Ein(e) <b>&quot;Schnellleser(in)&quot;</b> verschlingt Bücher und kann
+          sich trotzdem noch an Details erinnern. <br /> Eine echte{" "}
+          <b>Superpower</b> <br /> <b>Hast du sie auch?</b>
         </p>
-        <button>Mach den Test</button>
+        <LinkedButton
+          id={"start"}
+          buttonUrl={"test"}
+          content={"Mach den Test"}
+          icon={null}
```

Comment:
Make `icon` an optional type `?` so you don't need to pass down `null`

Reply:
So right! Thanks for helping me in reaching a clean code 🙏🏼